configs/ld/readme.md

```diff
@@ -0,0 +1,29 @@
+# Localization Distillation for Object Detection
+
+## Introduction
+
+LD is the extension of knowledge distillation on localization task, which utilizes the learned bbox distributions to transfer the localization dark knowledge from teacher to student.
+
+- [Localization Distillation for Object Detection](https://arxiv.org/abs/2102.12252)
+
+```latex
+@Article{zheng2021LD,
+  title={Localization Distillation for Object Detection},
+  author= {Zhaohui Zheng, Rongguang Ye, Ping Wang, Jun Wang, Dongwei Ren, Wangmeng Zuo},
+  journal={arXiv:2102.12252},
+  year={2021}
+}
+```
+
```

Comment:
suggest renaming the config like `ld_r50_gflv2_r101_...` and add a note to indicate the meaning of config file names.